DiscordPy/bot.py

```diff
@@ -70,11 +72,12 @@ async def getHelp(ctx, *arg):
 
 @bot.command(name='meme')
 async def getMeme(ctx, *arg):
-    if (len(imageList) == 0):
-        hot_python = subreddit.new(limit=20)
+    delta_time = datetime.datetime.now() - prev_time
+    delta_seconds = delta_time.total_seconds()
+    if (len(imageList) == 0 or delta_seconds >= 60):
+        hot_python = subreddit.new(limit=100)
         for i in hot_python:
-            extension = i.url[-3:]
-            if (extension == "jpg"):
+            if i.link_flair_text in {"Meme", "Loss", "Shitpost"} and i.url[-3:] in {"jpg", "png"}:

```

Comment:
Why are we taking the -3 index in the url?
Is this for image formats?
"JPEG" is 4 characters long, not 3. Does this mean we do not currently support JPEG fetching?

Reply:
Apparently the reddit images shorthand "JPEG" into "jpg". So the only 2 image format type extension is only 'jpg' and 'png'. It's just the way they name the image extensions :/

Reply:
Perfect!